routes/index.js

```diff
@@ -1,6 +1,6 @@
 module.exports = function(app, router){
-    app.use('/', require('./home.js')(router));
-    app.use('/', require('./user.js')(router));
-    app.use('/', require('./post.js')(router));
+    app.use('/api', require('./user.js')(router));
+    app.use('/api', require('./home.js')(router));
+    app.use('/api', require('./post.js')(router));
```

Comment:
Was worried about conflicts with my page, which is at /login. Thought it would also just be nice and semantic to add /api first. Can revert and change my route if needed.